test/crawler_web/features/authentication/register_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule CrawlerWeb.Authentication.RegisterTest do
+  use CrawlerWeb.FeatureCase, async: true
```

Comment:
For feature test, we prefer to run as it `synchronous` I'm not sure we have that on compass, but you can see the reason on https://github.com/nimblehq/nimblehq-web/pull/282/files#diff-d66aa29516732a764ecc3931bc058c70d74a61c1612739e3ffd5012521567bc3R491

![image](https://user-images.githubusercontent.com/11751745/206970035-4bfd4bd3-0d0d-42d3-a881-d9b592f8889f.png)

Reply:
Removed in [5939988](https://github.com/vnntsu/elixir-ic/pull/32/commits/5939988e956f649bc22c5b242696a2a6408c284c)

Reply:
Not remove it @vnntsu, let's set it as `, async: false` instead

Reply:
You make me remember the convention 🙏 set false instead of leaving empty. Updated in [f6b1690](https://github.com/vnntsu/elixir-ic/pull/32/commits/f6b1690d7869a11dc13023828d08bea472bd7831)